hnn_core/gui/gui.py

```diff
@@ -26,12 +26,92 @@
 from hnn_core.network import pick_connection
 from hnn_core.params import _extract_drive_specs_from_hnn_params
 from hnn_core.dipole import _read_dipole_txt
+from hnn_core.params_default import (get_L2Pyr_params_default,
+                                     get_L5Pyr_params_default)
 
 import base64
 import zipfile
 import numpy as np
 from copy import deepcopy
 
+cell_parameters_dict = {
```

Comment:
A suggestion for a future PR... Changing these to dictionaries with index at the keys and the tuples as the values would help with configuring and debugging the parameter mappings in the `update_*_cell_params` functions assuming there is a 1:1 order mapping of widgets to these lists.